db.go

```diff
@@ -14,18 +14,30 @@ const (
 
 var (
 	veryFarFuture = time.Date(3000, 1, 1, 1, 1, 1, 1, time.UTC)
+	dummyUsers map[UserId]*dummyUser
 )
 
+type UserId uint64
```

Comment:
Stick to int64 instead. Almost universally, if you see a database that produces autoincrementing rowids, the type of that field is int64 and not uint64.